README.md

```diff
@@ -547,12 +547,24 @@ signIn.renderEl({
 
 ### tokenManager.get
 
-Get a token that you have previously added to the `tokenManager` with the given `key`.
+Get a token that you have previously added to the `tokenManager` with the given `key`. The token object will be returned if it has not expired.
 
 - `key` - Key for the token you want to get
 
 ```javascript
-var token = signIn.tokenManager.get('my_id_token');
+signIn.tokenManager.get('my_id_token')
+.then(function(token) {
```

Comment:
nit: Lets be consistent with our promises in code blocks, where there are two spaces for the appended methods.